packages/react/src/components/Authenticator/shared/__tests__/__snapshots__/FormField.test.tsx.snap

```diff
@@ -40,7 +41,7 @@ exports[`FormField renders as expected in the unhappy path 1`] = `
   >
     <label
       class="amplify-label"
-      for="amplify-id-:r1:"
+      for="amplify-id-:r4:"
```

Comment:
Why did the `amplify-id:<ID>` values change?

Reply:
Think it's just the way they're generated with `useStableIdSpy.mockReturnValue('mock-id-0');`, if you change the ordering of the tests the id's change. I moved the new test to the bottom so it wouldn't change anything.